lib/input/helper/design_screen.dart

```diff
@@ -16,13 +16,18 @@ class DesignScreen implements DesignNodeFactory {
     DesignNode designNode,
     this.id,
     this.name,
+    this.type,
   }) {
     this.designNode = designNode;
   }
 
   Map<String, dynamic> toPBDF() {
     var result = <String, Object>{};
-    result['pbdfType'] = pbdfType;
+    if (this.type == 'symbolMaster') {
```

Comment:
Something seems weird here, don't know why we need this `if` statement, I imagine we should be passing the pbdfType down correctly all the time, is there a large refactor needed? It would clean this up.

Reply:
I agree, not sure why this is the case 🤔